cognito/mock/list_group_mocker.go

```diff
@@ -0,0 +1,36 @@
+package mock
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
+)
+
+func (m *CognitoIdentityProviderClientStub) AddListGroupWithName(name string) error {
+	newGroup, err := m.GenerateListGroup(name)
+	if err != nil {
+		return err
+	}
+	m.GroupsList = append(m.GroupsList, newGroup)
+	return nil
+}
```

Comment:
There was some other functions in that other file too, could you move them over?